### PR TITLE
Bump version v0.3.0 (3) to develop

### PR DESCRIPTION
**Pull request by sleepylee**
Release v0.2.0 (4) to master

### CODE REVIEW

**Comment by luongvo** [comment]
@sleepylee `flutter build` supports to set custom build number by github runner number, we don’t need to do it manually if we wanna build https://github.com/luongvo/flutter-survey/blob/develop/.github/workflows/deploy_staging.yml#L38. So this step is unnecessary I think

**Comment by sleepylee** [comment]
@luongvo but we are not having the CD pipeline ready 🤔  nor differentiate between bumped v0.2 or release v0.2 🤔  the build command currently doesn't take the build number input; but we actually don't plan to release anything from this `template` project to the Firebase anyway 🤔  perhaps I can just do the bump the version name for now 🤔

**Comment by luongvo** [comment]
Yes please, bumping the version name at the new development circle is enough

**Review by luongvo — APPROVED** [review]
ok, this should be merged after releasing 0.2.0 👌

File: pubspec.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ publish_to: 'none' # Remove this line if you wish to publish to pub.dev
 # In iOS, build-name is used as CFBundleShortVersionString while build-number used as CFBundleVersion.
 # Read more about iOS versioning at
 # https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html
-version: 0.2.0+3
+version: 0.3.0+4
```
Comment (sleepylee) [inline pubspec.yaml:18]: to differentiate between `0.2.0 (3)` - I now need to bump `0.3.0 (4)` 🤔  while I think this is still cumbersome as we discussed, I think we can retrace the release version later when we have the CD pipeline in hands 🤔

Comment (luongvo) [reply pubspec.yaml:18]: ok, so please update PR title as well

**Review by manh-t — APPROVED** [review]

**Pull request merged**